IdentityCore/src/oauth2/MSIDTokenResponse.m

```diff
@@ -126,6 +126,11 @@ + (MSIDProviderType)providerType
     @throw MSIDException(MSIDGenericException, @"Abstract method was invoked.", nil);
 }
 
+- (NSString *)accountIdentifier
+{
+    return self.idTokenObj.uniqueId;
```

Comment:
is this accountIdentifier different from uid+utid ?

Reply:
yes, uid+utid isn't part of OAuth protocol, it defines account id only AAD idp.